### PR TITLE
chore: add npm script to run WebdriverIO test

### PR DESCRIPTION
**Pull request by fasttime**
#### Prerequisites checklist

- [X] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[ ] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[X] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)

<s>Re-enabled WebdriverIO tests.</s> Added npm script `test:wdio` to run the WebdriverIO test locally.

#### Is there anything you'd like reviewers to focus on?

WebdriverIO has been running smoothly in CI for the last couple of months. Shall we try to re-enable it for tests in Jenkins? Or maybe just for local tests?

### CODE REVIEW

**Comment by aladdin-add** [comment]
it was disabled as encountered the error: `Command failed: which google-chrome`

https://github.com/eslint/eslint/issues/17572#issuecomment-1746824492

Is there any way we can verify that it has been fixed?

**Comment by fasttime** [comment]
> it was disabled as encountered the error: `Command failed: which google-chrome`
> 
> [#17572 (comment)](https://github.com/eslint/eslint/issues/17572#issuecomment-1746824492)
> 
> Is there any way we can verify that it has been fixed?

Hm, good point. I don't know how this can be verified. But if it's not necessary to run the browser tests in Jenkins maybe we could add a switch to the task in Makefile.js, so the browser test will run locally with `npm test`?

**Comment by mdjermanovic** [comment]
I think it's fine not to run the browser test on Jenkins, as we'll always have CI checks right before the release (when `@eslnt/js` changes are committed).

I also think it's fine to keep the browser test excluded from `npm test` because it rarely happens that a change breaks these tests and it would eventually be caught in CI anyway. We could add a package.json script that runs the browser test so that if the test happens to fail in CI, the contributor can easily run it locally.

**Comment by nzakas** [comment]
Yeah, we disabled these on Jenkins because there always seemed to be some configuration issue and we already knew that GitHub CI passed with it running. 

I also agree with @mdjermanovic that we don't need to add it to `npm test`. That command already take a long time to complete, and the browser use case is an edge case that we can watch for during CI.

**Comment by fasttime** [comment]
> I also think it's fine to keep the browser test excluded from `npm test` because it rarely happens that a change breaks these tests and it would eventually be caught in CI anyway. We could add a package.json script that runs the browser test so that if the test happens to fail in CI, the contributor can easily run it locally.

Thanks! I've re-excluded the browser test from the general test task and added an npm script to run it manually.

File: package.json [hunk 1]
```diff
@@ -32,7 +32,8 @@
     "test": "node Makefile.js test",
     "test:cli": "mocha",
     "test:fuzz": "node Makefile.js fuzz",
-    "test:performance": "node Makefile.js perf"
+    "test:performance": "node Makefile.js perf",
+    "test:wdio": "node Makefile.js wdio"
```
Comment (aladdin-add) [inline package.json:36]: wido is just the used tool, and the gha job name is `test-on-broswer`. :)
```suggestion
    "test:browser": "node Makefile.js wdio"
```

Comment (fasttime) [reply package.json]: Done in 9b57f7efa3d4bd687f29eb1e767025e26e681cf0, thanks.

**Review by mdjermanovic — APPROVED** [review]
LGTM, thanks!
